[RealProject] Deactivate .xsaccess

Currently the xsaccess can not be parsed.
As a workaround we need to ignore this error from the logs.
Deactivate the .xsaccess job parsing.


2021-08-25 10:46:38.681 [ERROR] [pool-8377-thread-1] o.e.d.c.s.api.AbstractSynchronizer - Resource [/registry/public/sap/slh/dcs/func/.xsaccess] skipped due to an error: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at line 3 column 21

--- a/modules/engines/engine-xssecurity/src/main/java/com/sap/xsk/xsaccess/ds/synchronizer/XSKSecuritySynchronizer.java
+++ b/modules/engines/engine-xssecurity/src/main/java/com/sap/xsk/xsaccess/ds/synchronizer/XSKSecuritySynchronizer.java
@@ -11,8 +11,6 @@
  */
 package com.sap.xsk.xsaccess.ds.synchronizer;
 
-import static java.text.MessageFormat.format;
-
 import com.sap.xsk.xsaccess.ds.api.IXSKAccessCoreService;
 import com.sap.xsk.xsaccess.ds.api.IXSKPrivilegeCoreService;
 import com.sap.xsk.xsaccess.ds.api.XSKAccessException;
@@ -36,7 +34,6 @@ import java.util.Set;
 import org.apache.commons.codec.digest.DigestUtils;
 import org.apache.commons.io.IOUtils;
 import org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizer;
-import org.eclipse.dirigible.core.scheduler.api.SchedulerException;
 import org.eclipse.dirigible.core.scheduler.api.SynchronizationException;
 import org.eclipse.dirigible.core.security.synchronizer.SecuritySynchronizer;
 import org.eclipse.dirigible.repository.api.IResource;
@@ -80,36 +77,36 @@ public class XSKSecuritySynchronizer extends AbstractSynchronizer {
    */
   @Override
   public void synchronize() {
-    synchronized (SecuritySynchronizer.class) {
-      if (beforeSynchronizing()) {
-        logger.trace("Synchronizing Privileges and Access artifacts...");
-        try {
-          startSynchronization(SYNCHRONIZER_NAME);
-          clearCache();
-          synchronizePredelivered();
-          synchronizeRegistry();
-          int immutablePrivilegesCount = PRIVILEGES_PREDELIVERED.size();
-          int immutableAccessCount = ACCESS_PREDELIVERED.size();
-
-          int mutablePrivilegesCount = PRIVILEGES_SYNCHRONIZED.size();
-          int mutableAccessCount = ACCESS_SYNCHRONIZED.size();
-          cleanup();
-          clearCache();
-          successfulSynchronization(SYNCHRONIZER_NAME,
-              format("Immutable: [Privileges: {0}, Access: {1}], Mutable: [Privileges: {2}, Access: {3}]",
-                  immutablePrivilegesCount, immutableAccessCount, mutablePrivilegesCount, mutableAccessCount));
-        } catch (Exception e) {
-          logger.error("Synchronizing process for Privileges and Access artifacts failed.", e);
-          try {
-            failedSynchronization(SYNCHRONIZER_NAME, e.getMessage());
-          } catch (SchedulerException e1) {
-            logger.error("Synchronizing process for Privileges and Access artifacts failed in registering the state log.", e);
-          }
-        }
-        logger.trace("Done synchronizing Privileges and Access artifacts.");
-        afterSynchronizing();
-      }
-    }
+//    synchronized (SecuritySynchronizer.class) {
+//      if (beforeSynchronizing()) {
+//        logger.trace("Synchronizing Privileges and Access artifacts...");
+//        try {
+//          startSynchronization(SYNCHRONIZER_NAME);
+//          clearCache();
+//          synchronizePredelivered();
+//          synchronizeRegistry();
+//          int immutablePrivilegesCount = PRIVILEGES_PREDELIVERED.size();
+//          int immutableAccessCount = ACCESS_PREDELIVERED.size();
+//
+//          int mutablePrivilegesCount = PRIVILEGES_SYNCHRONIZED.size();
+//          int mutableAccessCount = ACCESS_SYNCHRONIZED.size();
+//          cleanup();
+//          clearCache();
+//          successfulSynchronization(SYNCHRONIZER_NAME,
+//              format("Immutable: [Privileges: {0}, Access: {1}], Mutable: [Privileges: {2}, Access: {3}]",
+//                  immutablePrivilegesCount, immutableAccessCount, mutablePrivilegesCount, mutableAccessCount));
+//        } catch (Exception e) {
+//          logger.error("Synchronizing process for Privileges and Access artifacts failed.", e);
+//          try {
+//            failedSynchronization(SYNCHRONIZER_NAME, e.getMessage());
+//          } catch (SchedulerException e1) {
+//            logger.error("Synchronizing process for Privileges and Access artifacts failed in registering the state log.", e);
+//          }
+//        }
+//        logger.trace("Done synchronizing Privileges and Access artifacts.");
+//        afterSynchronizing();
+//      }
+//    }
   }
 
   /**

